vllm/core/interfaces.py

```diff
@@ -40,6 +40,11 @@ def get_block_space_manager_class(version: str):
                 EmbeddingModelBlockSpaceManager)
             return EmbeddingModelBlockSpaceManager
 
+        if version == "simple":
+            from vllm.core.simple_model_block_manager import (
+                SimpleModelBlockSpaceManager)
+            return SimpleModelBlockSpaceManager
+
```

Comment:
Can we remove the "embedding" and "simple" versions and instead return the class directly inside `ModelMode`? (Only keep "v1" and "v2" since they are actually versions)

Reply:
Edit: Actually I think this function isn't used anymore?

Reply:
removed.